docs/sources/developers/plugins/sign-a-plugin.md

```diff
@@ -32,7 +32,16 @@ To verify ownership of your plugin, you need to generate an API key that you'll
 
 Public plugins need to be reviewed by the Grafana team before you can sign them.
 
-1. Submit your plugin for [review]({{< relref "publish-a-plugin/#publishing-your-plugin-for-the-first-time" >}})
+1. Submit your plugin for review. 
+
+   1. Sign in to [grafana.com](https://grafana.com)
+   1. On the My Account page, go to Org Settings.
```

Comment:
'click' instead of 'go to'?

Reply:
I prefer _go to_ here. _Click_ locks the instructions into a narrower interaction pattern: it implies that Org settings are a button (which may change in the future) and that this button needs a mouse (and I'm pretty sure that today the resource could be accessed with keyboard navigation).

So IMO, even though _go to_ is less precise, it keeps it a little more future proof and accessible.

I got this idea from the Google style guide:

> When practical, state instructions in terms of what the user should accomplish, rather than focusing on the widgets and gestures. By avoiding reference to UI elements, you help the user understand the purpose of an instruction, and it can help future-proof procedures. [^1]

[^1]: https://developers.google.com/style/ui-elements

Reply:
Maybe it's neither click or go to. Note that Org Settings is not a link that is clicked upon, it is a drop-down menu that is toggled. (Sorry for the confusion. This text was supplied by a developer, and the page looks different today than when I wrote this.)  I think this is how it needs to be fixed:

1. On the My Account page, select your organization in the Organization menu at the top left.
1. On the Organization's page, click **My plugins**.